app_dart/lib/src/service/firestore.dart

```diff
@@ -111,38 +112,42 @@ class FirestoreService {
 }
 
 /// Generates task documents based on targets.
-List<Document> targetsToTaskDocuments(Commit commit, List<Target> targets) {
-  final Iterable<Document> iterableDocuments = targets.map(
-    (Target target) => Document(
-      name: '$kDatabase/documents/$kTaskCollectionId/${commit.sha}_${target.value.name}_1',
-      fields: <String, Value>{
-        kTaskCreateTimestampField: Value(integerValue: commit.timestamp!.toString()),
-        kTaskEndTimestampField: Value(integerValue: kTaskDefaultTimestampValue.toString()),
-        kTaskBringupField: Value(booleanValue: target.value.bringup),
-        kTaskNameField: Value(stringValue: target.value.name),
-        kTaskStartTimestampField: Value(integerValue: kTaskDefaultTimestampValue.toString()),
-        kTaskStatusField: Value(stringValue: Task.statusNew),
-        kTaskTestFlakyField: Value(booleanValue: false),
-        kTaskCommitShaField: Value(stringValue: commit.sha),
-      },
+List<firestore.Task> targetsToTaskDocuments(Commit commit, List<Target> targets) {
```

Comment:
nit: is there any way to consolidate the naming of TaskDocument and Task? Not a blocker, but the naming could be slightly confusing in the future.

Reply:
Yeah, there will be a clean up of these namings after we fully switched to firestore. Added a todo in https://github.com/flutter/flutter/issues/142951 to note.